tests/clang_native.py

```diff
@@ -5,22 +5,37 @@
 
 import logging
 import os
+import platform
+import sys
 from tools.shared import PIPE, run_process, CLANG_CC, CLANG_CXX
 from tools.utils import MACOS, WINDOWS, path_from_root
 
 logger = logging.getLogger('clang_native')
 
+def get_native_triple():
+  arch = {
+      'arm64': 'arm64',
```

Comment:
should be `aarch64`

Reply:
Oh, on MacOS it's `arm64` but clang accepts both. So maybe we need both this existing line and the one you suggested.